server/services/utils/formValidation.ts

```diff
@@ -147,7 +143,16 @@ const validationProcedures = {
     getErrorMessage: (fieldConfig, errorPath) => getIn(fieldConfig, [...errorPath, 'validationMessage']),
   },
   additional: {
-    getSchema: () => additionalConditionsSchema,
+    getSchema: ({ version }) => {
```

Comment:
There's at least 3 representations of additional config:
* Form config
* Validation messages
* Validation schema

Here it reads the version from the form config to work out which schema to use.